src/System.Private.ServiceModel/tests/Scenarios/Infrastructure/WcfFactTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Reviewers: this test case was included mostly to illustrate usage and to allow me to test.  I will delete it from the PR before merging.